test-parachains/adder/build.rs

```diff
@@ -20,6 +20,6 @@ fn main() {
 	build_current_project_with_rustflags(
 		"wasm_binary.rs",
 		WasmBuilderSource::Crates("1.0.7"),
-		"-C link-arg=--import-memory",
+		"-C link-arg=--export=__heap_base",
```

Comment:
I'm very bad at understanding those flags what has this changed ?

Reply:
Good that you found this. I wanted to open an issue for this ;) Before we provided the memory, now the WASM needs to provide it (currently default for our runtimes). However, we need to make sure that we at least provide 1GB of memory to the validation.